flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -190,6 +208,19 @@ class IOSFlutterLocalNotificationsPlugin
         'initialize', initializationSettings.toMap());
   }
 
+  /// Requests given permission from user and returns current permission status.
+  Future<bool> requestPermissions({
+    bool requestSoundPermission,
```

Comment:
Sorry just realised this now but given the method itself is called `requestPermissions`, the name of the arguments can simplified to `sound`, `alert` and `badge`.

Reply:
TBH, I would also remove permission parameters from `IOSInitializationSettings`, but this would be a breaking change.

Reply:
For now I went with shorter parameter names